logging.tmux

```diff
@@ -5,24 +5,21 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "$CURRENT_DIR/scripts/variables.sh"
 source "$CURRENT_DIR/scripts/shared.sh"
 
+
 setup_logging_key_binding() {
-	local key="$(get_tmux_option "$logging_key_option" "$default_logging_key")"
-	tmux bind-key "$key" run-shell "$CURRENT_DIR/scripts/toggle_logging.sh"
+	tmux bind-key "$logging_key" run-shell "$CURRENT_DIR/scripts/toggle_logging.sh"
 }
 
 setup_screen_capture_key_binding() {
-	local key="$(get_tmux_option "$pane_screen_capture_key_option" "$default_pane_screen_capture_key")"
-	tmux bind-key "$key" run-shell "$CURRENT_DIR/scripts/screen_capture.sh"
+	tmux bind-key "$pane_screen_capture_key" run-shell "$CURRENT_DIR/scripts/screen_capture.sh"
 }
 
 setup_save_complete_history_key_binding() {
-	local key="$(get_tmux_option "$save_complete_history_key_option" "$default_save_complete_history_key")"
-	tmux bind-key "$key" run-shell "$CURRENT_DIR/scripts/save_complete_history.sh"
+	tmux bind-key "$save_complete_history_key" run-shell "$CURRENT_DIR/scripts/save_complete_history.sh"
 }
 
 setup_clear_history_key_binding() {
-	local key="$(get_tmux_option "$clear_history_key_option" "$default_clear_history_key")"
-	tmux bind-key "$key" run-shell "$CURRENT_DIR/scripts/clear_history.sh"
+	tmux bind-key "$clear_history_key" run-shell "$CURRENT_DIR/scripts/clear_history.sh"
```

Comment:
Given that all the above functions are just one-liners now, how do you feel about just having them in the `main` function?

Reply:
It's a good suggestion! I will make it in the second commit :)